config/routes.rb

```diff
@@ -63,8 +63,8 @@
   end
 
   mount Peek::Railtie => "/peek"
+  mount Blazer::Engine, at: "blazer"
   if Rails.env.development?
     mount LetterOpenerWeb::Engine => "letter_opener"
-    mount Blazer::Engine, at: "blazer"
```

Comment:
let's leave this as a dev-only feature, and copy prod db to local env when needed. that keeps blazer security issues to a minimum.